src/tokens/Assets.tsx

```diff
@@ -483,6 +484,14 @@ function AssetsScreen({ navigation, route }: Props) {
 
 AssetsScreen.navigationOptions = {
   ...headerWithBackButton,
+  headerRight: HeaderRight,
+}
+
+function HeaderRight() {
+  if (getFeatureGate(StatsigFeatureGates.SHOW_IMPORT_TOKENS_FLOW)) {
+    return <ImportTokenButton />
+  }
+  return <></>
```

Comment:
what is the advantage of using this over `null`?

Reply:
agree, returning null or undefined is preferable i think